cmd/list.go

```diff
@@ -57,4 +62,5 @@ func init() {
 	listCmd.Flags().BoolVar(&installed, "installed", false, "list the installed Terraform versions")
 	listCmd.Flags().BoolVar(&aliases, "aliases", false, "list the aliased Terraform versions")
 	listCmd.Flags().IntVar(&maxResults, "max-results", 500, "maximum number of versions to list")
+	listCmd.Flags().BoolVarP(&includePreReleaseVersions, "pre-release", "p", false, "include pre-release versions")
```

Comment:
We're not using any short-hand flags yet, maybe just use a `BoolVar` for now and we can implement short-hand flags throughout the tool later?

Reply:
Addressed in a84cc21ef9b32076fe173272c161ad01b3d264a2